packages/fondue/src/components/InlineDialog/InlineDialog.tsx

```diff
@@ -4,14 +4,17 @@ import { Overlay } from '@utilities/dialogs/Overlay';
 import { Z_INDEX_MODAL } from '@utilities/dialogs/constants';
 import { type WithRequired } from 'src/types/withRequired';
 
-import { type BaseDialogProps, Modality, type OverlayProps } from '../../types';
+import { Modality, type BaseDialogProps, type OverlayProps } from '../../types';
 
 export type InlineDialogProps = Omit<
     OverlayProps,
     'theme' | 'isDetached' | 'verticalAlignment' | 'withArrow' | 'arrowCustomColors' | 'shadow' | 'isDialog'
 > &
     Omit<BaseDialogProps, 'darkUnderlay'> & { unsafe_zIndex?: number };
 
+/**
+ * @deprecated Please use updated button component from `@frontify/fondue/components`.
+ */
```

Comment:
It is the flyout component, also let's add the migration guide link
```suggestion
/**
 * @deprecated Use `Flyout` from `@frontify/fondue/components` instead. Also check {@link https://github.com/Frontify/fondue/blob/main/packages/components/MIGRATING.md#flyout the migration guide}.
 */
```

Reply:
changed